### PR TITLE
Read document id from a file

### PR DESCRIPTION
**Pull request by takecare**
#### Problem

We need to avoid hardcoding the document ID due to security concerns, as we only want a specific set of people to have that ID.

#### Solution

Use the solutions already in place for this, i.e., `*CredentialsReader`. This means the document ID will be stored in a properties file and its contents will be read when necessary. As we're git ignoring `*.credentials` files the file containing the ID won't be published to the repo.

##### Test(s) added

Yes.

Added assertJ as a test dependency 'cause it allows for more readable tests. I intend on updating some other tests (unrelated with this PR) later.

### CODE REVIEW

File: sheets/src/main/java/com/novoda/github/reports/sheets/properties/DocumentIdReader.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package com.novoda.github.reports.sheets.properties;
+
+import com.novoda.github.reports.properties.PropertiesReader;
+
+public class DocumentIdReader {
+
+    private static final String SHEETS_PROPERTIES_FILENAME = "sheets.credentials";
+    private static final String DOCUMENT_ID_KEY = "DOCUMENT_ID";
+
+    private PropertiesReader propertiesReader;
```
Comment (frapontillo) [inline sheets/src/main/java/com/novoda/github/reports/sheets/properties/DocumentIdReader.java]: Good ol' PropertiesReader.

File: sheets/src/test/java/com/novoda/github/reports/sheets/network/SheetsServiceClientTest.java [hunk 2]
```diff
@@ -1,35 +1,41 @@
...
 import static org.mockito.Matchers.anyString;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
 import static org.mockito.MockitoAnnotations.initMocks;
 
 public class SheetsServiceClientTest {
 
+    private static final String ANY_DOCUMENT_ID = "1rMeGnlugO312to0loBwN3x0QTvAxoHwv4Pe_SYXR1YE";
```
Comment (frapontillo) [inline sheets/src/test/java/com/novoda/github/reports/sheets/network/SheetsServiceClientTest.java]: So we just moved the secret to a test file? 😆

Comment (takecare) [inline sheets/src/test/java/com/novoda/github/reports/sheets/network/SheetsServiceClientTest.java]: well, this is not really the secret as another document will be used

File: sheets/src/main/java/com/novoda/github/reports/sheets/properties/DocumentIdReader.java (hunk 1 above)
Comment (takecare) [inline sheets/src/main/java/com/novoda/github/reports/sheets/properties/DocumentIdReader.java]: `PropertiesReader` is love, `PropertiesReader` is life

**Pull request merged**